cmd/manager/main.go

```diff
@@ -152,9 +152,23 @@ func main() {
 		os.Exit(1)
 	}
 
-	// Add the Metrics Service and Service Monitor
-	if err := winmetrics.Add(ctx, cfg, namespace); err != nil {
-		log.Error(err, "failed to add Metrics Service and Service Monitor")
+	// Get metrics configuration
+	metricsConfig, err := winmetrics.NewMetricsConfig(cfg, namespace)
```

Comment:
Nit: I think we can get rid of `winmetrics` naming. We had to use that because of the SDK `metrics` pkg which has now been removed.